src/app/integrations/detail-page/detail.component.ts

```diff
@@ -402,4 +408,12 @@ export class IntegrationsDetailComponent extends IntegrationViewBase
     this.integrationSubscription.unsubscribe();
     this.routeSubscription.unsubscribe();
   }
+
+  exportIntegration() {
```

Comment:
I think this should probably go into the integration-support service...

Reply:
Yeah. probably should.